jwt-bcp/draft-sheffer-oauth-jwt-bcp.md

```diff
@@ -191,19 +325,42 @@ may sign JWTs with different "typ" values:
 * "JWT+set+oidc-logout" for OID Connect Logout events.
 
 Specifications that define JWT applications SHOULD define a unique "typ" value.
+-->
+
+Each application of JWTs defines a profile specifying the required and optional JWT claims
+and the validation rules associated with them.
+To prevent substitution of JWTs from one context into another, a number of strategies may be employed:
```

Comment:
As long as each distinct application area employs a simple and valid strategy, things are fine.  It's only if the same issuer (with the same signing keys) starts issuing multiple kinds of JWTs for different purposes that confusion is even possible.  We should be clear about that context in any recommendations that we write.